torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -159,8 +160,19 @@ def _create_op_overload_to_exporter_key_table() -> (
     # TODO(justinchuby): Improve how the table is constructed.
     table: Dict[Union[torch._ops.OpOverload, Callable], str] = {}
 
+    # Some ops in `torch.ops.aten` are not discoverable through `dir(torch.ops.aten)`,
+    # but retrievable via explicit lookup.
+    # https://github.com/pytorch/pytorch/issues/99681
+    # This is a workaround to make sure we register ONNX symbolic functions for these.
+    onnx_supported_aten_lookup_table = [
+        k.split("::")[1] for k in _ATENLIB_FUNCTIONS.keys() if k.startswith("aten::")
+    ]
+
     for op_namespace in (torch.ops.aten, torch.ops.prims):
-        for attr_name in dir(op_namespace):
+        attr_names = dir(op_namespace)
+        if op_namespace is torch.ops.aten:
+            attr_names += onnx_supported_aten_lookup_table
+        for attr_name in attr_names:
             op_overload_packet = getattr(op_namespace, attr_name)
```

Comment:
Should we handle when attr_name is not in op_namespace?

Reply:
No, it will crash. Can be an assertion but I was being lazy since we have hand written mapping table and this is caught at loading for now.

Reply:
At this point I won't update in this PR, unless a merge conflict happens with #99284 :)

Reply:
Cool. I will wait for this to land first. Although there could be an issue when torch decides to remove ops

Reply:
Oh interesting, are you thinking about relaxing the constraint? My thought was to reveal this as soon as possible as CI breaks. Assertion just makes it explicit.

Reply:
I am undecided. Letting it break makes sense too considering we want to know asap when something like this happens